docs/index.rst

```diff
@@ -24,6 +24,7 @@ and development.
 
    baseplate-healthcheck: Is your service alive? <cli/healthcheck>
    baseplate-serve: The application server <cli/serve>
+   baseplate-script: Run offline jobs <cli/script>
```

Comment:
Could this be used to run something like consumers (e.g. the email s3->thrift streamer)?  If so, the word "offline" is a bit confusing to me here.